langchain/chains/sql_database/base.py

```diff
@@ -11,7 +11,8 @@
 from langchain.llms.base import BaseLLM
 from langchain.prompts.base import BasePromptTemplate
 from langchain.sql_database import SQLDatabase
-
+#TODO: decide whether this is neccesary
+from exception_handler import ExceptionHandler
 
 class SQLDatabaseChain(Chain, BaseModel):
     """Chain for interacting with SQL Database.
```

Comment:
Add a `max_retries` optional argument alongside others like `llm` and `database`

Reply:
Default to `max_retries = 1` so that by default chain does not retry call